### ISSUE TITLE
Use neuron=*=mpi*

### ISSUE THREAD

**Issue by espenhgn** [conda-forge/lfpy-feedstock#2]
I would like to use `neuron=*=mpi*` (or `neuron=*=mpi_mpich*` for now) for LFPy, but couldn't yet figure out how to set this up in the `recipe/meta.yaml` in PR #1. Users of LFPy should not try and use the LFPy.Network class with the serial Neuron build (it will work most likely, but cannot benefit on multicore systems). No additional dependencies will be installed anyway, so I don't see the problem in enforcing the parallel Neuron build.

Creating a new env and running the LFPy tests with the mpich-enabled Neuron package seems fine on both macos and ubuntu: 
```
$ conda create -n lfpy lfpy neuron=*=mpi*
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /Users/ehagen/anaconda3/envs/lfpy

  added / updated specs:
    - lfpy
    - neuron[build=mpi*]

The following NEW packages will be INSTALLED:
...
  neuron             conda-forge/osx-64::neuron-7.6.7-mpi_mpich_py37hc2cee5e_2
...

$ conda activate lfpy
$ python -c "import LFPy; LFPy.run_tests()"
Mechanisms already loaded from path: /Users/ehagen/anaconda3/envs/lfpy/lib/python3.7/site-packages/LFPy/test.  Aborting.
...

test LFPy.Cell class and methods:
.........................................................................
----------------------------------------------------------------------
Ran 73 tests in 20.320s

OK
...
```

**Comment by espenhgn** [comment]
My recipe still leaves me confused. Running `conda-build .` w. 6ea5a892ce679cec7208f4c82f40b94cdfdeaba6 apparently works fine in a new conda environment with `conda-smithy` (Ubuntu), but fails on Azure: 
```
Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 445, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 436, in execute
    verify=args.verify, variants=args.variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2299, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1445, in build
    cwd=src_dir, stats=build_stats)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 374, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 354, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-e', '/home/conda/feedstock_root/build_artifacts/lfpy_1559037820480/work/conda_build.sh']' returned non-zero exit status 1.
##[error]Bash exited with code '1'.
```

**Comment by espenhgn** [comment]
Fixed by #1. Closing.
